src/utils.js

```diff
@@ -74,15 +74,20 @@ function createCSVDownload(columns, data, options) {
     ? options.onDownload(buildHead, buildBody, columns, data)
     : `${CSVHead}${CSVBody}`.trim();
 
-  if (options.onDownload && csv === false) {
-    return;
+  const downloadHasBeenAborted = csv === false;
```

Comment:
I'm not sure that assigning to a variable here that is only used once just below is clearer than using `options.onDownload`. This part wasn't broken before, so I think let's just leave it as it was for now.